### PR TITLE
#167313366: commenting on an article.

### PR DESCRIPTION
**Pull request by habinezadalvan**
#### What does this PR do?
- Creates and read a comment on a particular article
#### Description of Task to be completed?
- Comment on an article 
- Read all comment of a particular article
#### How should this be manually tested?
- Clone the repo
- run **git checkout ft-comment-on-an-article-167313366**
- run **npm install**
- run **npm run devserver**
- Use postman
Method: POST
- url **http://localhost:3000/api/v1/comments/articleSlug**
- In the json form use {"body": "comment"}
#### Any background context you want to provide?
- React, redux
#### What are the relevant pivotal tracker stories?
[#167313366](https://www.pivotaltracker.com/story/show/167313366)
#### Screenshots (if appropriate)
Screen Shot 2019-10-31 at 11 18 33
Screen Shot 2019-10-31 at 11 17 55

#### Questions:
none

### CODE REVIEW

File: src/redux/actions/actionTypes.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const SIGNUP_ERROR = 'SIGNUP_ERROR';
 export const VIEW_ARTICLE_SUCCESS = 'VIEW_ARTICLE_SUCCESS';
 export const VIEW_ARTICLE_ERROR = 'VIEW_ARTICLE_ERROR';
 export const BASE_URL = 'https://codepirates-ah-backend.herokuapp.com';
-export const BACKEND_URL = 'https://codepirates-ah-backend-staging.herokuapp.com';
+export const BACKEND_URL = 'http://localhost:3000';
```
Comment (SokoPaulSokool) [inline src/redux/actions/actionTypes.js]: put back the backend url.

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]

File: src/redux/actions/actionTypes.js (hunk 1 above)
Comment (habinezadalvan) [reply src/redux/actions/actionTypes.js]: I worked on that.

File: src/components/commentArticle/LikeDislikeReplies/LikeDislikeReplies.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { mount } from 'enzyme';
+// import { Provider } from 'react-redux';
+// import { MemoryRouter } from 'react-router-dom';
+// import mockedStore from '../../../__mocks__/store';
```
Comment (SokoPaulSokool) [inline src/components/commentArticle/LikeDislikeReplies/LikeDislikeReplies.test.js]: Commented code

File: src/components/commentArticle/commentCard/commentCard.test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      username: 'userone',
+      image: 'image',
+      commentId: 0,
+      childrenComments: mockedChildrenComments,
+    });
+    // const textArea = wrapper.find('textarea');
+    // textArea.simulate('click', {});
+    // textArea.simulate('change', {});
+    // const button = wrapper.find('#button');
+    // button.simulate('click', {});
```
Comment (SokoPaulSokool) [inline src/components/commentArticle/commentCard/commentCard.test.js]: commented code

File: src/components/commentArticle/commentTextArea/commentTextArea.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      font-weight: bold;
+    }
+  }
+  textarea {
+    border: 4px solid red;
+    // width: 100%;
+    // border-radius: 10px;
+    // outline: none;
+    // padding-left: 10px;
+    // border: 1px solid rgba(0, 0, 0, 0.2);
```
Comment (SokoPaulSokool) [inline src/components/commentArticle/commentTextArea/commentTextArea.scss]: commented code

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]

File: src/components/commentArticle/commentTextArea/commentTextArea.scss (hunk 4 above)
Comment (habinezadalvan) [reply src/components/commentArticle/commentTextArea/commentTextArea.scss]: I removed them.

File: src/components/commentArticle/commentCard/commentCard.test.js (hunk 3 above)
Comment (habinezadalvan) [reply src/components/commentArticle/commentCard/commentCard.test.js]: I removed them

File: src/components/commentArticle/LikeDislikeReplies/LikeDislikeReplies.test.js (hunk 2 above)
Comment (habinezadalvan) [reply src/components/commentArticle/LikeDislikeReplies/LikeDislikeReplies.test.js]: I removed them

**Pull request merged**